Sample Data/CitrixAnalytics_indicatorSummary_CL.json

```diff
@@ -0,0 +1,29 @@
+[{
```

Comment:
As per the schema, it should have risk_probability_s field, but sample data does not have the data for that column. Since this is missing, workbook queries are failing which are using this column. This is the same for all other columns. Please provide sample data with all the columns.